urdf/uw_lidar_robot.xacro

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
+
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="uw_lidar">
+  <xacro:arg name="inertial_reference_frame" default="world"/>
+  <xacro:include filename="$(find nps_uw_sensors_gazebo)/urdf/uw_lidar.xacro"/>
+
+  <link name="robot_base_link"/>
+
+  <link name="robot_link">
+    <inertial>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <mass value="200"/>
+      <inertia
+        ixx="96.67" ixy="0.0" ixz="0.0"
+        iyy="158.17" iyz="0.0"
+        izz="146.83"/>
+    </inertial>
+    <visual>
+      <geometry>
+        <mesh filename="package://nps_uw_sensors_gazebo/meshes/eNOVUS.dae"/>
```

Comment:
I don't think this mesh should be in the sensors package.    Having a demo of the lidar on a robot is good, but it should be in the dave repository.  This mesh is renamed and included in the dave repository as the caldus ROV

Reply:
This is a legacy of the first cut DVL demo and tutorial that Michael started from.  The mesh was there for the sole purpose of a "real" robot to mount the DVL (and now the UW LIDAR) on without introducing any dependencies (the dave repo version is in the caldus_description package).  It would be easy enough to replace with basic shape or even a dimensionless link.  Alternatively, we could probably replace that xacro snippet with a call to one of the vehicle exemplars.

Reply:
Since the purpose of this demo was just to show how the UWL could be mounted to a vehicle and we now have another demo that does that in the Dave repository, we don't need this any more. I've deleted it and updated the test instructions to refer to uw_lidar_standalone.launch, which shows the lidar in an empty tank, similar to the previous release.